v1/types/types.gunk

```diff
@@ -326,4 +336,43 @@ type Profile struct {
 
 	// Contact address
 	ContactAddress Address `pb:"13" json:"contact_address"`
+
+	// profile number
+	ProfileNUmber string `pb:"14" json:"profile_number"`
+
+	// Face image of the customer
+	FaceImageUrl string `pb:"15" json:"face_image_url"`
+
+	// Date when the face image was added/updated
+	FaceImageDate string `pb:"16" json:"face_image_date"`
+
+	// RelationshipStatus. Ex: Single | Married
+	RelationshipStatus string `pb:"17" json:"relationship_status"`
+
+	// Number of dependents
+	Dependants int `pb:"18" json:"dependants"`
```

Comment:
should be dependents I think?

Reply:
Yup, looks like I have commented it correctly.